java-checks/src/test/files/checks/security/EmptyDatabasePasswordCheck.java

```diff
@@ -0,0 +1,48 @@
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import static org.sonar.java.checks.security.EmptyDatabasePasswordCheckTest.EMPTY_PASSWORD;
+import static org.sonar.java.checks.security.EmptyDatabasePasswordCheckTest.NON_EMPTY_PASSWORD;
+
+class S2115 {
+  void foo(Properties connectionProps) throws SQLException {
```

Comment:
There is no testing of the highlighting of the error. I think we should add one.